test/Microsoft.AspNetCore.SignalR.Client.Tests/TestHttpMessageHandler.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.AspNetCore.SignalR.Client.Tests
     public class TestHttpMessageHandler : HttpMessageHandler
     {
         private Func<HttpRequestMessage, CancellationToken, Task<HttpResponseMessage>> _handler;
+        public bool IsFirstRequest = true;
```

Comment:
`IsFirstLongPoll` or something. There are various kinds of requests handled by this handler :)